crates/driver/src/infra/config/file/mod.rs

```diff
@@ -306,8 +306,8 @@ pub enum UniswapV3Preset {
 }
 
 mod uniswap_v3 {
-    pub fn default_max_pools_to_initialize() -> u64 {
-        50
+    pub fn default_max_pools_to_initialize() -> usize {
+        100
```

Comment:
intended?

Reply:
Yes, the default we use in the current driver is `100` so I wanted to make that the same so we don't get surprised when we switch over to the new driver.